docs/source/configuration.rst

```diff
@@ -679,6 +679,17 @@ Data in localStorage on the other hand is kept indefinitely.
     roster contact statuses will not become out of sync in a single session,
     only across more than one session.
 
+synchronize_presence
+--------------------
+
+* Default: ``true``
+
+Valid options: ``true``, ``false``, ``a resource name``.
+
+This option determines how to synchronize presence with other own client resources.
+If set to ``true``, converse.js synchronize presence with other own client resources.
+If set to ``false``, converse.js doesn't synchronize presence with other own client resources.
+If set to ``a resource name``, converse.js synchronize presence with only clients that have particular resource name.
 
```

Comment:
Some language suggestions: 

> This option determines how to synchronize presence with other own client resources.

This option lets you synchronize your chat status (`online`, `busy`, `away`) with other chat clients. In other words, if you change your status to `busy` in a different chat client, your status will change to `busy` in converse.js as well.

> If set to ``true``, converse.js synchronize presence with other own client resources.

If set to ``true``, converse.js will synchronize with all other clients you are logged in with.

> If set to ``false``, converse.js doesn't synchronize presence with other own client resources.

If set to ``false``, this feature is disabled.

> If set to ``a resource name``, converse.js synchronize presence with only clients that have particular resource name.

If set to ``a resource name``, converse.js will synchronize only with a client that has that particular resource assigned to it.